gateway-service/src/main/java/org/zowe/apiml/gateway/config/WebSecurity.java

```diff
@@ -333,6 +334,22 @@ public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http, Au
             .build();
     }
 
+    @Bean
```

Comment:
Any reason why this couldn't be included in the securityWebFilterChain @Order(1) ?

Reply:
I'm not sure, this is not my change, I guess it was copied from v2, but on the other hand I suppose this makes it a bit more readable as opposed to having one big filter chain definition

Reply:
We don't have many definitions so far so I'm ok with that, it was just because we are adding only one endpoint, and the authentication type is the same. But we can keep it separately.

Reply:
yes for now it's one endpoint until next PR where I want to check these authentication configuration, it will have the exceptions for application/version /health and so on

Reply:
Okok!